inspirehep/modules/crossref/config.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""CrossRef Configuration."""
+
+from __future__ import absolute_import, division, print_function
+
+
+response_code = {'success': 200,
```

Comment:
Constants should be in `ALL_CAPS_SEPARATED_BY_UNDERSCORES`, and if you use the `init_config` pattern described below this could become `CROSSREF_RESPONSE_CODES`.

On the other hand there's no need to obsess particularly about this, as we already know that this dictionary is going to  disappear soon with the better approach. Read this piece of documentation and try to figure out yourself how it should be: http://flask.pocoo.org/docs/0.12/api/#flask.Flask.make_response